docker/aligned_base.Dockerfile

```diff
@@ -41,6 +41,8 @@ FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
 
 FROM chef AS planner
 
+ENV RUSTFLAGS="-C codegen-units=256 -C incremental=true -C opt-level=0"
```

Comment:
Remove the incremental flag. It doesn't really help in CI environments and generates a lot of data for the cache.

Reply:
Also, you might want to use `lto=false`.

Reply:
It is caching the files though

Reply:
Also lto = false is the default of release

Reply:
I removed the incremental, cargo chef mentions it's redudant and increases the amount of layers in docker file. lto false seems to not change anything since the default is release.